modules/Bio/EnsEMBL/Variation/Pipeline/PhenotypeAnnotation/BasePhenotypeAnnotation.pm

```diff
@@ -1406,47 +1439,86 @@ sub _get_set_ids {
 
 }
 
+sub _clean_phenotype_desc {
+  my ($self, $description, $type) = @_;
+
+  $description =~ s/^\s+|\s+$//g; # Remove spaces at the beginning and the end of the description
+  $description =~ s/\n//g; # Remove 'new line' characters
+  $description =~ s/[\(\)]//g; # Remove characters ( )
+
+  # Replace special characters in the phenotype description
+  $description = replace_char($description);
+  $description = replace_hex($description);
+
+  if($type) {
+    $description = lc($description);
+    $description =~ s/\“//g;
+    $description =~ s/\”//g;
+    $description =~ s/\s+/ /g; # Remove extra space
+
+    # fix some typos
+    $description =~ s/sjoegren larsson syndrom$/sjogren larsson syndrome/;
+    $description =~ s/sjorgren larrson syndrome/sjogren larsson syndrome/;
+    $description =~ s/marinesco sjoegren syndrome/marinesco sjogren syndrome/;
```

Comment:
Suggesting you use "-" because thats the disease names so `marinesco-sjoegren syndrome and sjoegren-larsson syndrome`
Also typo in L1460 ` sjoegren larsson syndrom$ >  sjoegren-larsson syndrome`

Reply:
Thanks for the suggestions!
L1460 is not a typo, I found an example with `syndrom` instead of `syndrome` but it has been fixed in G2P.